### PR TITLE
lint files

### PR DESCRIPTION
**Pull request by phapdinh**
This fixes issue issue #4415

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ module.exports = {
       'requireReturnForObjectLiteral': false
     }],
 
-    'arrow-parens': ['error', 'always'],
+    // 'arrow-parens': ['error', 'always'],
```
Comment (phapdinh) [inline .eslintrc.js]: I commented out this rule because sometimes arrow functions if they have only one parameter don't need parens

Comment (jdalton) [reply .eslintrc.js]: It's fine to enable. I want to be consistent and always use parens.

Comment (phapdinh) [reply .eslintrc.js]: I have added back this rule

**Comment by jdalton** [comment]
Thank you @phapdinh!

**Pull request merged**